src/components/views/right_panel/UserInfo.tsx

```diff
@@ -448,7 +448,15 @@ export const UserOptionsSection: React.FC<{
                     const inviter = new MultiInviter(roomId || "");
                     await inviter.invite([member.userId]).then(() => {
                         if (inviter.getCompletionState(member.userId) !== "invited") {
-                            throw new Error(inviter.getErrorText(member.userId) ?? undefined);
+                            const errorStringFromInviterUtility = inviter.getErrorText(member.userId);
+                            if (errorStringFromInviterUtility) {
+                                throw new Error(errorStringFromInviterUtility);
+                            } else {
+                                throw new UserFriendlyError(
+                                    `User (%(user)s) did not end up as invited to %(roomId)s but no error was given from the inviter utility`,
+                                    { user: member.userId, roomId, cause: undefined },
+                                );
+                            }
```

Comment:
The [remaining TS strict errors](https://github.com/matrix-org/matrix-react-sdk/actions/runs/4535420375/jobs/7990846522?pr=10440) are unrelated and tracked by https://github.com/vector-im/element-web/issues/24899

And there are some nits from files we didn't even change